src/components/request/useRequestTableData.tsx

```diff
@@ -38,9 +40,11 @@ function useRequestTableData({
   timestamp,
   ancillaryData,
   requestTxHash,
+  rules,
 }: UseRequestTableParams) {
   const [rows, setRows] = useState<IRow[]>([]);
   const [headerCells] = useState<ICell[]>(hc);
+  rules = useOptimisticGovernorRules();
```

Comment:
this should not be overriding a property that is passed in, assume they are read only.  What you need to do is hoist this call into the parent component and pass in the result if you want to keep this class "pure".  the other option is to remove the rules property and define the variable here

Reply:
I removed the override and instead defined the variable in the update.